modules/web/src/app/shared/components/application-list/add-application-dialog/template.html

```diff
@@ -116,17 +116,17 @@
               </div>
             </div>
             <mat-form-field>
-              <mat-label>Namespace</mat-label>
+              <mat-label>Application Installation Namespace</mat-label>
```

Comment:
isn't better to have the name of the application first ?

Reply:
Since Application Installation Namespace has higher scope than other fields so I thought it might be better to put it first. Also, I didn't want the confusion between both namespace fields by putting them together.